<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_username">
       <property name="text">
        <string>username: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineedit_username">
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>2-10 characters</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_password">
       <property name="text">
        <string>password: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TLineEdit" name="lineedit_password">
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>5-10 characters</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_login">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>log in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_signup">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>sign up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string>close</string>
       </property>
       <property name="icon">
        <iconset resource="../../resource/images/images.qrc">
         <normaloff>:/images/close.svg</normaloff>:/images/close.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_accept">
       <property name="text">
        <string>试用</string>
       </property>
       <property name="icon">
        <iconset resource="../../resource/images/images.qrc">
         <normaloff>:/images/accessibility.svg</normaloff>:/images/accessibility.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_22">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resource/images/images.qrc">:/images/22open.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_33">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resource/images/images.qrc">:/images/33open.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">frontend/templatewidget/tlineedit/tlineedit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
